zello-sdk/src/com/zello/sdk/Sdk.java

```
@@ -240,6 +232,41 @@ void onPause() {
 
 	//region Zello SDK Methods
 
+	//region Permissions
+
+	void requestVitalPermissions() {
+		Context context = _context;
+		if (context != null) {
+			try {
+				Intent intent = new Intent();
+				intent.setComponent(new ComponentName(_package, _pttInvisibleActivityClass));
+				intent.putExtra(Constants.EXTRA_REQUEST_VITAL_PERMISSIONS, true);
+				context.startActivity(intent);
+			} catch (Exception ignored) {
+				// ActivityNotFoundException
```

How will the client know if this were to silently fail?